php/libraries/NDB_BVL_InstrumentStatus.class.inc

```diff
@@ -262,12 +262,12 @@ class NDB_BVL_InstrumentStatus
     /**
      * Sets the current validity status of the instrument
      *
-     * @param string $status the new status from the set
-     *                       (NULL, 'Questionable', 'Invalid', 'Valid')
+     * @param ?string $status the new status from the set
+     *                        (NULL, 'Questionable', 'Invalid', 'Valid')
      *
      * @return void
      */
-    function setValidityStatus(string $status): void
+    function setValidityStatus(?string $status): void
```

Comment:
as stated in the description of the function, `$status` should be allowed to be `null`. These changes are necessary to accept null as a valid status.